File: problemas/1100/1144.js
const n = require('fs').readFileSync('/dev/stdin', 'utf8');

for (let i = 1; i <= n; i++) {
  const quadrado = Math.pow(i, 2);
  const cubo = Math.pow(i, 3);
  console.log(`${i} ${quadrado} ${cubo}`);
  console.log(`${i} ${quadrado + 1} ${cubo + 1}`);
}

File: problemas/1100/1144.dart
import 'dart:io';
import 'dart:math';

void main() {
  final n = int.parse(stdin.readLineSync());
  for (int i = 1; i <= n; i++) {
    final quadrado = pow(i, 2);
    final cubo = pow(i, 3);
    stdout.writeln('$i $quadrado $cubo');
    stdout.writeln('$i ${quadrado + 1} ${cubo + 1}');
  }
}